src/shared.ts

```diff
@@ -81,7 +81,7 @@ export function filterOptions(
 export function getActionFromKey(
 	event: KeyboardEvent,
 	menuOpen: boolean,
-	isCurrentOptionDisabled: boolean
+	isCurrentOptionDisabled = false
```

Comment:
I think we'd still want a type on this param. Could we do: 
`isCurrentOptionDisabled: boolean = false` instead?

Reply:
Barys pointed me at the Typescript default param docs - nicely done!